NgocMaiDang_democypress/cypress/e2e/registrationform_spec.cy.js

```diff
@@ -0,0 +1,39 @@
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // returning false here prevents Cypress from
+  // failing the test
+  return false
+});
+
+describe('Verify successful submission of Registration Form', () => {
+  it('Should fill out the Registration form and submiss successfully', () => {
```

Comment:
This form has 2 happy case:
- Submit with all fields input
- Submit with mandatory fields input